[inductor][Observability] Add log for Optimus to enable easier debug

mengluy0125 opened the pull request:
Summary: The log breaks one of ads-model export flows, and we change the log to debug

Test Plan: see details in D49710166

Differential Revision: D49844303




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

jackiexu1992 reviewed (approved):
FYI there is a lint test failure. Not sure its a blocking issue or not. May check with pytorch team.

mengluy committed:
[inductor][Observability] Add log for Optimus to enable easier debug (#110452)

Summary:
The log breaks one of ads-model export flows, and we change the log to debug


Test Plan:
see details in D49710166





cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

Reviewed By: dshi7, jackiexu1992

Differential Revision: D49844303

Pulled By: mengluy0125
